es-design-system/pages/organisms/es-support-card.vue

```diff
@@ -0,0 +1,369 @@
+<template>
+    <div>
+        <h1 class="mb-300">
+            Support card
+        </h1>
+        <div class="mb-450">
+            <h2>
+                Default
+            </h2>
+            <es-support-card
+                image-alt-text="Energy advisor team"
+                :image-src="eaMultiple"
+                primary-cta-text="Schedule a call"
+                primary-cta-url="https://www.energysage.com">
+                <template #headline>
+                    <span class="align-items-center d-flex text-success">
+                        <icon-verified
+                            class="mr-50"
+                            height="22px"
+                            width="22px" />
+                    </span>
+                    We're here to help
+                </template>
+                <template #description>
+                    Our Energy Advisors provide expert, unbiased energy advice at no cost to you.
```

Comment:
Can we swap this with lorem-ipsum or more generic copy?

Reply:
Done